enterprise/frontend/src/embedding-sdk/hooks/private/use-init-data.unit.spec.tsx

```diff
@@ -36,17 +39,17 @@ import { createMockState } from "metabase-types/store/mocks";
 
 const TEST_USER = createMockUser();
 
-const TestComponent = ({ config }: { config: SDKConfig }) => {
+const TestComponent = ({ authConfig }: { authConfig: MetabaseAuthConfig }) => {
   const dispatch = useSdkDispatch();
 
   const loginStatus = useSdkSelector(getLoginStatus);
   const isLoggedIn = useSdkSelector(getIsLoggedIn);
 
   useInitData({
-    config: {
-      ...config,
+    authConfig: {
```

Comment:
could we use `createMockAuthProviderUriConfig` here too since we're already using it in the test scenarios?